EAP-TLS_Solution/EAP-TLS Client/tests/tests.c

```diff
@@ -0,0 +1,496 @@
+#include <errno.h>
```

Comment:
Oh I see, maybe it is "tests.c" rather than "test.c".  Either way, the readme file has both versions, so it needs to be corrected.